test/Table_Tests/src/In_Memory/Table_Conversion_Spec.enso

```diff
@@ -0,0 +1,158 @@
+            Table.from_objects uniform_json ["first", "last", "age"] . should_equal expected
+
+        Test.specify "Generates a table from a vector of JSON objects, with missing fields" <|
+            expected = Table.from_rows ["last", "age"] [["Smith", 23], ["Burton", 34]]
+            Table.from_objects uniform_json ["last", "age"] . should_equal expected
+
+        Test.specify "Generates a table from a vector of JSON objects, with extra fields" <|
+            expected = Table.from_rows ["first", "middle", "last", "age"] [["Mary", Nothing, "Smith", 23], ["Joe", Nothing, "Burton", 34]]
+            Table.from_objects uniform_json ["first", "middle", "last", "age"] . should_equal expected
+
+        Test.specify "Generates a table from a vector of JSON objects, with missing and extra fields" <|
+            expected = Table.from_rows ["first", "middle", "age"] [["Mary", Nothing, 23], ["Joe", Nothing, 34]]
+            Table.from_objects uniform_json ["first", "middle", "age"] . should_equal expected
+
+    Test.group "from_objects with non-uniform JSON vector" <|
+        Test.specify "Generates a table from a non-uniform vector of JSON objects" <|
+            expected = Table.from_rows ["first", "last", "age", "height", "weight"] [["Mary", "Smith", 23, Nothing, Nothing], [Nothing, Nothing, Nothing, 1.9, 70]]
+            Table.from_objects non_uniform_json . should_equal expected
+
+        Test.specify "Generates a table from a non-uniform vector of JSON objects, with exact fields" <|
+            expected = Table.from_rows ["first", "last", "age", "height", "weight"] [["Mary", "Smith", 23, Nothing, Nothing], [Nothing, Nothing, Nothing, 1.9, 70]]
+            Table.from_objects non_uniform_json ["first", "last", "age", "height", "weight"] . should_equal expected
+
+        Test.specify "Generates a table from a non-uniform vector of JSON objects, with missing fields" <|
+            expected = Table.from_rows ["last", "weight"] [["Smith", Nothing], [Nothing, 70]]
+            Table.from_objects non_uniform_json ["last", "weight"] . should_equal expected
+
+        Test.specify "Generates a table from a non-uniform vector of JSON objects, with extra fields" <|
+            expected = Table.from_rows ["first", "middle", "last", "age", "height", "weight"] [["Mary", Nothing, "Smith", 23, Nothing, Nothing], [Nothing, Nothing, Nothing, Nothing, 1.9, 70]]
+            Table.from_objects non_uniform_json ["first", "middle", "last", "age", "height", "weight"] . should_equal expected
+
+        Test.specify "Generates a table from a non-uniform vector of JSON objects, with missing and extra fields" <|
+            expected = Table.from_rows ["first", "middle", "height", "weight"] [["Mary", Nothing, Nothing, Nothing], [Nothing, Nothing, 1.9, 70]]
+            Table.from_objects non_uniform_json ["first", "middle", "height", "weight"] . should_equal expected
```

Comment:
I'd consider a test case where we specify only fields from one of these values, i.e. only `["first"]`.

but not super important

Reply:
Done